dist/common/scripts/scylla_util.py

```diff
@@ -427,6 +421,80 @@ def getEphemeralOsDisks(self):
+
+
+# {
+#     "dataDisks": [
+#       {
+#         "caching": "None",
+#         "createOption": "Empty",
+#         "diskSizeGB": "1024",
+#         "image": {
+#           "uri": ""
+#         },
+#         "lun": "0",
+#         "managedDisk": {
+#           "id": "/subscriptions/xxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxx/resourceGroups/macikgo-test-may-23/providers/Microsoft.Compute/disks/exampledatadiskname",
+#           "storageAccountType": "Standard_LRS"
+#         },
+#         "name": "exampledatadiskname",
+#         "vhd": {
+#           "uri": ""
+#         },
+#         "writeAcceleratorEnabled": "false"
+#       }
+#     ],
+#     "imageReference": {
+#       "id": "",
+#       "offer": "UbuntuServer",
+#       "publisher": "Canonical",
+#       "sku": "16.04.0-LTS",
+#       "version": "latest"
+#     },
+#     "osDisk": {
+#       "caching": "ReadWrite",
+#       "createOption": "FromImage",
+#       "diskSizeGB": "30",
+#       "diffDiskSettings": {
+#         "option": "Local"
+#       },
+#       "encryptionSettings": {
+#         "enabled": "false"
+#       },
+#       "image": {
+#         "uri": ""
+#       },
+#       "managedDisk": {
+#         "id": "/subscriptions/xxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxx/resourceGroups/macikgo-test-may-23/providers/Microsoft.Compute/disks/exampleosdiskname",
+#         "storageAccountType": "Standard_LRS"
+#       },
+#       "name": "exampleosdiskname",
+#       "osType": "Linux",
+#       "vhd": {
+#         "uri": ""
+#       },
+#       "writeAcceleratorEnabled": "false"
+#     }
+# }
```

Comment:
Please explain what this comment means so somebody looking at it in 4 months doesn't have to guess.

Reply:
removing, it was just a placeholder for the refactoring

Reply:
will remove